modules/010-user-authn-crd/crds/group.yaml

```
@@ -0,0 +1,76 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: groups.deckhouse.io
+  labels:
+    heritage: deckhouse
+    module: user-authn
+spec:
+  group: deckhouse.io
+  scope: Cluster
+  names:
+    plural: groups
+    singular: group
+    kind: Group
+  preserveUnknownFields: false
+  versions:
+    - name: v1alpha1
+      served: true
+      storage: true
+      subresources:
+        status: {}
+      schema:
+        openAPIV3Schema:
+          type: object
+          description: |
+            Contains information about a user group.
+          required:
+            - spec
+          properties:
+            spec:
+              type: object
+              required:
+                - name
+                - members
+              properties:
+                name:
+                  type: string
```

```suggestion
                  type: string
                  description: Unique group name.
```
In case the field stays here (see the comment above)